src/ontology/edits/oeo-tasks.omn

```diff
@@ -0,0 +1,709 @@
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determination method for wind characteristics of a region is a methodology with the goal to understand the specific wind characteristics of a predefined region, encompassing data acquisition, modelling, and analytical procedures.",
+        rdfs:label "determination method for wind characteristics of a region"@en
+    
+    SubClassOf: 
+        OEO_00020166,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00390004
+    
+    
+Class: OEO_00390003
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Data acquisition and pre-processing is an action specification that describes the steps the bearer takes to acquire raw data and how this data is cleaned, transformed, and structured for subsequent analysis.",
+        rdfs:label "data acquisition and pre-processing"@en
+    
+    SubClassOf: 
+        OEO_00390035
+    
+    
+Class: OEO_00390004
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determined wind characteristics of a region is an object specification that defines the goal of a methodology as a state of understanding the specific wind characteristics of a predefined region.",
+        rdfs:label "determined wind characteristics of a region"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/IAO_0000005>
+    
+    
+Class: OEO_00390005
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Extrapolate wind speed vertically to hub height is an action specification that describes how wind speed at the hub height of a wind turbine is inferred from measurements taken at a different height.",
+        rdfs:label "extrapolate wind speed vertically to hub height"@en
+    
+    SubClassOf: 
+        OEO_00390035
+    
+    
+Class: OEO_00390007
+
+    Annotations: 
+        OEO_00020426 "Add:
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determination process for wind characteristics of a region is a process that follows a determination methodology for wind characteristics to gain knowledge of the specific wind characteristics of a predefined region.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Determination process for wind characteristics of a region is a process that follows a determination methodology for wind characteristics to gain knowledge of the specific wind characteristics of a predefined region.",
```
Please add to to do list in Issue: 
add axiom to `study region`/`region of relevance`. Maybe harmonize the wording in the definitions, too.